src/node.cc

```diff
@@ -350,6 +350,14 @@ MaybeLocal<Value> RunBootstrapping(Environment* env) {
           .IsNothing())
     return MaybeLocal<Value>();
 
+  // Make sure that no request or handle is created during bootstrap -
+  // if necessary those should be done in pre-exeuction.
+  // TODO(joyeecheung): print these requests/handles before aborting
```

Comment:
I think you can use `PrintLibuvHandleInformation` for this without having to do much extra work

Reply:
Thanks, that'll do for handles at least